common/src/main/java/com/wynntils/core/chat/tabs/ChatTabModel.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.core.chat.tabs;
+
+import com.wynntils.core.managers.Model;
+import com.wynntils.mc.event.RenderChatEvent;
+import com.wynntils.mc.utils.McUtils;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import net.minecraft.client.gui.components.ChatComponent;
+import net.minecraft.network.chat.Component;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+public class ChatTabModel extends Model {
+    private static final ChatComponent EMPTY_CHAT_COMPONENT = new ChatComponent(McUtils.mc());
+
+    private static ChatTab focusedTab = null;
+
+    private static Map<ChatTab, ChatComponent> chatTabData = Map.of();
+
+    public static void init() {
+        chatTabData = new ConcurrentHashMap<>();
```

Comment:
Why not assign this from the start? Is `init` called multiple times if this is enabled/disabled?

Reply:
Yes. And we want to re-assign it so in-case we disable the model, we don't hold onto useless chat data in memory.

Reply:
But you do ` chatTabData.clear()` in `disable`, is that not enough?